exporter/exporter.yaml

```diff
@@ -1 +1,37 @@
-topics: {}
\ No newline at end of file
+kafka-brokers: localhost:9092
```

Comment:
please remove this.. this is an opensource project and the config file should not be filled in..